assignments/prototypes.js

```diff
@@ -32,7 +62,22 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+Humanoid.prototype = Object.create(GameObject.prototype);
```

Comment:
you don't need to do this. You chained CharacterStats to GameObject prototype already, so if you chain Humanoid to CharacterStats, you get Humanoid chained up to GameObject for free. The way you're doing it, you are overwriting the link you make to CharacterStats on line 65 with a brand new link on line 66